tests/test_json_schema_validation.py

```diff
@@ -0,0 +1,35 @@
+"""This tests the loading of schema files and working validation of a JSON file against a schema"""
+
+import os
+from unittest import TestCase
+
+from validator.json_schema_validation import load_arrayexpress_submission_schema, validate_submission_json
+
+
+class TestLoadingSchema(TestCase):
+
+    def test_load_submission_schema(self):
+        # Call schema loading function that reads the schema files in a fixed location
+        schema_file, schema = load_arrayexpress_submission_schema()
+        # Check that we have non null output
+        self.assertIsNotNone(schema_file)
+        self.assertIsNotNone(schema)
+
+    def test_reading_schemas(self):
+        # Call the schema loading function
+        schema_file, schema = load_arrayexpress_submission_schema()
+        # Check the schema content can be parsed
+        self.assertIn("samples", schema["properties"])
+
+
+class TestValidateJson(TestCase):
+
+    def setUp(self):
+        # This test uses a very simple test JSON and test schema
+        wd = os.path.dirname(os.path.realpath(__file__))
```

Comment:
if `os.path` is the only thing you'd be using from `os`, you might as well be explicit in the import and then save some characters here.

Reply:
OK, done.